### DISCUSSION TITLE
Welcome to SimpleWebsockets.jl Discussions!

### DISCUSSION THREAD

**Discussion by citkane** [citkane/SimpleWebsockets.jl#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by citkane** [comment]
@zahachtah

Hi Jon,

Hope that the Julia websockets are being useful for you.
Thanks for release 0.2.2

I have a time window that is allowing me to look at code again, and I have done some housekeeping to this repo. To let you know what has happened:

- There were some issues in my test code, as well as some breaking changes to packages involved in testing
- Because tests were failing, automatic updates with [juliaregistrator](https://github.com/JuliaRegistries/Registrator.jl) failed
- juliaregistrator needs to have sequential releases, so I could not jump to 0.2.3 over your 0.2.2
- I deleted your 0.2.2 and created a new one

My 0.2.2 incorporates the latest version of all dependencies, so you should be good to go with it from JuliaHub without pulling directly from my GIT.

If your workflow is purely inter-process (not web to process) communication, you may find [ZeroMQ](https://zeromq.org/) interesting.
It is one of those kind of magic projects that makes the thought of the power of the internet interesting again...

All the best,
Michael

**Reply by markusic-gh** [reply]
I upgraded to Julia 1.9.0 and SimpleWebsockets appears to no longer work.  For example I get:

LoadError: ConnectError: HTTP.Exceptions.ConnectError

I'm not savvy enough with websockets to understand the issue.

**Comment by citkane** [comment]
Hi @markusic-gh

I had a quick look at this using Julia 1.9 and all the [Basic Usage](https://github.com/citkane/SimpleWebsockets.jl#basic-usage-server) examples are working as expected.

The tests are however hanging for SSL connections..
If you comment out [this line](https://github.com/citkane/SimpleWebsockets.jl/blob/3548e94943f0cbd6a2189d4548775e06215c7a1d/test/runtests.jl#L38), the tests all pass.

SimpleWebsockets does not implement HTTP or SSL, so it looks like it is going to be a dependency issue.

Can you please confirm if you are using SSL?
